src/chaat/config.clj

```diff
@@ -0,0 +1,28 @@
+(ns chaat.config
+  (:require [clojure.java.io :as io]
+            [clojure.edn :as edn])
+  (:import (java.io PushbackReader)))
+
+;; change defn's -> def's, not much of an advantage here
```

Comment:
[minor] Does this comment still need to be here? Can we address it right away?

Reply:
Done, took care of this (: